src/components/common/SearchBar/searchBar.tsx

```diff
@@ -192,6 +193,7 @@ const SearchBar = ({
 
   //update parent and queries
   function onSelect_internal(newValue: SearchQuery[]) {
+    setErrorTooltip(!newValue.length); //Check if tooltip needs to be displayed
```

Comment:
the tool tip never goes away on the homepage, not sure why, seems this should do it.